api/controllers/adminController.js

```diff
@@ -52,13 +51,15 @@ class AdminController {
     _prepareToSend(adminSaved) {
         if (Array.isArray(adminSaved)) {
             let admins = util.copyObject(adminSaved);
-            for (const admin in admins) {
-                admins[admin].password = null;
+            for (const admin of admins) {
+                Reflect.deleteProperty(admin, 'password');
```

Comment:
Correct me if I'm wrong, but this is the equivalent of a "Reflection"? If so, isn't this kind of a bad practice?

Reply:
this is it or "delete admin.password", which the lint says is not recomendable and suggests to do this way